libs/stats/odc/stats/_pq.py

```diff
@@ -21,9 +21,12 @@
     "thin cirrus",
 )
 
-# filters = list of iterable tuples of morphological operations in the order you want them performed,
-# e.g. [[("opening", 2), ("dilation", 5)]]
-default_filters = [[("opening", 2), ("dilation", 5)], [("opening", 0), ("dilation", 5)]]
+# filters - dict of band name and list of iterable tuples of morphological operations
+#           in the order you want them to perform
+default_filters = [
+    {"clear_2_5": [("opening", 2), ("dilation", 5)]},
+    {"clear_0_5": [("opening", 0), ("dilation", 5)]}
+]
```

Comment:
```suggestion
default_filters = {
    "clear_2_5": [("opening", 2), ("dilation", 5)],
    "clear_0_5": [("opening", 0), ("dilation", 5)]
}
```
No need to wrap that in a list, sorry I must have been unclear.

We just want a dictionary, for every key we compute a new "quality" band with different morphological filters applied.

Reply:
ok, I will give it one more try